tests/screenshot/apps/components/VirtualGridList.js

```diff
@@ -27,15 +26,9 @@ const updateDataSize = (dataSize) => {
 		const
 			count = ('00' + i).slice(-3),
 			caption = `Item ${count}`,
-			color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
-			label = `SubItem ${count}`,
-			src = {
-				hd: `http://placehold.it/200x200/${color}/ffffff&text=Image ${i}`,
-				fhd: `http://placehold.it/300x300/${color}/ffffff&text=Image ${i}`,
-				uhd: `http://placehold.it/600x600/${color}/ffffff&text=Image ${i}`
-			};
```

Comment:
I hope some image is shown.

How about

import img from '../../images/600x600.png';
src={img}

Reply:
Suggested changes.